`npm i` 경보 고쳐

```
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from the root project
npm WARN   36 more (@testing-library/react, react-dom, antd, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.0.0" from rc-overflow@1.0.1
npm WARN node_modules/rc-overflow
npm WARN   rc-overflow@"^1.0.0" from rc-select@12.0.4
npm WARN   node_modules/rc-select
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react-dom@17.0.1
npm WARN node_modules/react-dom
npm WARN   react-dom@"^17.0.1" from the root project
npm WARN   30 more (@testing-library/react, antd, @ant-design/react-slick, ...)
```

--- a/.github/workflows/firebase-hosting-merge.yml
+++ b/.github/workflows/firebase-hosting-merge.yml
@@ -11,8 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: npm i
-      - run: npm run build
+      - run: yarn install
+      - run: yarn build
       - uses: FirebaseExtended/action-hosting-deploy@v0
         with:
           repoToken: "${{ secrets.GITHUB_TOKEN }}"

--- a/.github/workflows/firebase-hosting-pull-request.yml
+++ b/.github/workflows/firebase-hosting-pull-request.yml
@@ -8,8 +8,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: npm i
-      - run: npm run build
+      - run: yarn install
+      - run: yarn build
       - uses: FirebaseExtended/action-hosting-deploy@v0
         with:
           repoToken: "${{ secrets.GITHUB_TOKEN }}"

--- a/.github/workflows/lint.yaml
+++ b/.github/workflows/lint.yaml
@@ -8,5 +8,5 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: "12"
-      - run: npm install
+      - run: yarn install
       - run: npx eslint src --ext .js,.jsx,.ts,.tsx

--- a/.github/workflows/prettier.yaml
+++ b/.github/workflows/prettier.yaml
@@ -8,5 +8,5 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: "12"
-      - run: npm install
+      - run: yarn install
       - run: npx prettier --check src

--- a/.github/workflows/unit-tests.yaml
+++ b/.github/workflows/unit-tests.yaml
@@ -8,5 +8,5 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: "12"
-      - run: npm install
-      - run: npm test
+      - run: yarn install
+      - run: yarn test

